header.html

```diff
@@ -1,77 +1,77 @@
+          <li><a href="/7-Logical-Agents">7. Logical Agents</a></li>
+          <li><a href="/8-First-Order-Logic">8. First Order Logic</a></li>
+          <li><a href="/9-Inference-In-First-Order-Logic">9. Inference in First Order Logic</a></li>
+          <li><a href="/10-Classical-Planning">10. Classical Planning</a></li>
+          <li><a href="/11-Planning-And-Acting-In-The-Real-World">11. Planning and Acting in the Real World</a></li>
+          <li><a href="/12-Knowledge-Representation">12. Knowledge Representation</a></li>
         </ul>
       </li>
       <li class="dropdown">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#">Part 4
         <span class="caret"></span></a>
         <ul class="dropdown-menu">
-          <li><a href="../13-Quantifying-Uncertainity">13. Quantifying Uncertainty</a></li>
-          <li><a href="../14-Probabilistic-Reasoning">14. Probabilistic Reasoning</a></li>
-          <li><a href="../15-Probabilistic-Reasoning-Over-Time">15. Probabilistic Reasoning Over Time</a></li>
-          <li><a href="../16-Making-Simple-Decisions">16. Making Simple Decisions</a></li>
-          <li><a href="../17-Making-Complex-Decisions">17. Making Complex Decisions</a></li>
+          <li><a href="/13-Quantifying-Uncertainity">13. Quantifying Uncertainty</a></li>
+          <li><a href="/14-Probabilistic-Reasoning">14. Probabilistic Reasoning</a></li>
+          <li><a href="/15-Probabilistic-Reasoning-Over-Time">15. Probabilistic Reasoning Over Time</a></li>
+          <li><a href="/16-Making-Simple-Decisions">16. Making Simple Decisions</a></li>
+          <li><a href="/17-Making-Complex-Decisions">17. Making Complex Decisions</a></li>
         </ul>
       </li>
       <li class="dropdown">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#">Part 5
         <span class="caret"></span></a>
         <ul class="dropdown-menu">
-          <li><a href="../18-Learning-From-Examples">18. Learning from Examples</a></li>
-          <li><a href="../19-Knowledge-In-Learning">19. Knowledge in Learning</a></li>
-          <li><a href="../20-Learning-Probabilistic-Models">20. Learning Probabilistic Models</a></li>
-          <li><a href="../21-Reinforcement-Learning">21. Reinforcement Learning</a></li>
+          <li><a href="/18-Learning-From-Examples">18. Learning from Examples</a></li>
+          <li><a href="/19-Knowledge-In-Learning">19. Knowledge in Learning</a></li>
+          <li><a href="/20-Learning-Probabilistic-Models">20. Learning Probabilistic Models</a></li>
+          <li><a href="/21-Reinforcement-Learning">21. Reinforcement Learning</a></li>
         </ul>
       </li>
       <li class="dropdown">
         <a class="dropdown-toggle" data-toggle="dropdown" href="#">Part 6
         <span class="caret"></span></a>
         <ul class="dropdown-menu">
-          <li><a href="../22-Natural-Language-Processing">22. Natural Language Processing</a></li>
```

Comment:
Why is this needed? Having the relative URLs help us decouple this from deployment?

Reply:
I removed this when I split the section into two pages originally. Relative url assumes we only inject the header in pages one directory down which breaks links if we try to add more pages per section. Some operations in my section require the site to be on a server to work so we can't really avoid decoupling...